### PR TITLE
Support multiple ACM hubs

### PR DESCRIPTION
**Pull request by clacroix12**

### CODE REVIEW

File: ocs_ci/framework/__init__.py [hunk 1]
```diff
@@ -216,11 +216,19 @@ def switch_ctx(self, index=0):
...
 
-    def get_acm_index(self):
+    def get_active_acm_index(self):
+        for cluster in self.clusters:
+            if cluster.MULTICLUSTER["active_acm_cluster"]:
+                return cluster.MULTICLUSTER["multicluster_index"]
+        # if no active cluster is found, designate one
+        return self.designate_active_acm_cluster()
+
+    def designate_active_acm_cluster(self):
```
Comment (petr-balogh) [inline ocs_ci/framework/__init__.py:228]: Add docstring please

File: ocs_ci/framework/conf/default_config.yaml [hunk 2]
```diff
@@ -298,3 +298,4 @@ MULTICLUSTER:
   multicluster_index: 0
   acm_cluster: False
   primary_cluster: False
+  active_acm_cluster: False
```
Comment (petr-balogh) [inline ocs_ci/framework/conf/default_config.yaml:301]: add to documentation in conf/README.md

File: ocs_ci/framework/__init__.py (hunk 1 above)
Comment (clacroix12) [reply ocs_ci/framework/__init__.py:234]: Added docstring for this function and `get_active_acm_index`.

**Review by shylesh — APPROVED** [review]
LGTM

**Pull request merged**